Websites/webkit.org/wp-content/plugins/webkit-survey/plugin.php

```diff
@@ -158,6 +157,7 @@ public static function calculate_results() {
             return unserialize($cached);
 
         $responses = get_post_custom_values($key);
+        if (!$responses) $responses = [];
```

Comment:
so in JS you could have this be something like
```js
responses = get_post_custom_values(key) || [];
```
does PHP have a similar thing?

Reply:
No, or at least it hasn't had. I haven't kept up with the latest syntax updates.